CHANGELOG.md

```diff
@@ -60,6 +61,8 @@ List of incompatible ABI changes in this release:
 * avifEncoderSetCodecSpecificOption() now returns avifResult instead of void to
   report memory allocation failures.
 * At decoding, avifIOStats now returns the same values as at encoding.
+* avifRGBImageAllocatePixels() now returns avifResult instead of void to report
+  memory allocation failures.
```

Comment:
Optional: Now that the function returns avifResult, it could also report invalid input parameters, so this bullet point could be written more generally. But what you have is just fine because it reflects the current implementation.